app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,232 @@
+        app:layout_constraintStart_toEndOf="@id/button_zero" />
+
+    <Button
+        android:id="@+id/button_add"
+        style="@style/button_style"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_between_button_xsmall"
+        android:layout_marginEnd="@dimen/margin_between_button_xsmall"
+        android:layout_marginBottom="@dimen/margin_bottom_button_small"
+        android:text="@string/activity_main_calculator_add_text"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/button_equals" />
+
+    <Button
+        android:id="@+id/button_equals"
+        style="@style/button_style"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:layout_marginStart="@dimen/margin_between_button_xsmall"
+        android:layout_marginEnd="@dimen/margin_between_button_xsmall"
+        android:layout_marginBottom="@dimen/margin_bottom_button_small"
+        android:text="@string/activity_main_calculator_equals_text"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/button_add"
+        app:layout_constraintStart_toEndOf="@id/button_clean" />
+
+    <ImageView
+        android:id="@+id/image_view_icon_calculator"
+        android:layout_width="@dimen/icon_size_xbig"
+        android:layout_height="@dimen/icon_size_xbig"
+        android:layout_marginStart="@dimen/margin_start_icon_small"
+        android:contentDescription="@string/content_description_icon_calculator"
+        android:src="@drawable/activity_main_calculator_icon_drawable"
+        app:layout_constraintBottom_toTopOf="@id/guideline_title"
+        app:layout_constraintEnd_toStartOf="@id/text_view_title"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/text_view_title"
+        style="@style/title_style"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:text="@string/activity_main_calculator_title_text"
+        app:layout_constraintBottom_toTopOf="@id/guideline_title"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/image_view_icon_calculator"
         app:layout_constraintTop_toTopOf="parent" />
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <TextView
+        android:id="@+id/text_view_result"
+        style="@style/result_style"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_marginBottom="@dimen/margin_bottom_result_xbig"
```

Comment:
you can put this property into the result_style

Reply:
this still here

Reply:
Solved!